README.md

```diff
@@ -78,6 +90,18 @@ There are several Haskell formatters available that auto-format .hs files. I rec
 
   (again, this will take a while)
 
+- **Macbook Note:** If you are using a MacBook with M1 Apple Chip stack may not work for you. In that case use 
+
+  `~$ stack install stylish-haskell`
```

Comment:
Should this be `~$ cabal install stylish-haskell`?

Reply:
hi @iburzynski , changed